src/serialization.cpp

```diff
@@ -104,39 +104,39 @@ Status serializeBlobToTensorProto_2(
             networkOutput->getName(), networkOutput->getPrecision());
         return StatusCode::INTERNAL_ERROR;
     }
-    switch (networkOutput->getPrecision()) {
-    case InferenceEngine::Precision::FP32:
+    switch (networkOutput->getPrecision_2()) {
+    case ovms::Precision::FP32:
```

Comment:
Couldn't we replace this switch with usage of:
` tensorflow::DataType TensorInfo::getPrecisionAsDataType(Precision precision)` ?

Reply:
Or at least simplify part of it